lib/sourceParams.js

```diff
@@ -75,6 +77,8 @@ module.exports = function(app) {
     cssCompiledOutput: params.staticsRoot + (params.cssCompiledOutput || pkg.rooseveltConfig.cssCompiledOutput || '.build/css'),
     cssCompilerWhitelist: params.cssCompilerWhitelist || pkg.rooseveltConfig.cssCompilerWhitelist || undefined,
     jsPath: params.staticsRoot + (params.jsPath || pkg.rooseveltConfig.jsPath || 'js'),
+    bundledJsPath: params.staticsRoot + (params.bundledJsPath || pkg.rooseveltConfig.bundledJsPath || 'js/.bundled'),
```

Comment:
It might be nice to concat the `jsPath` along with the `staticsRoot`. So a user can just specify a name of the js file they want without having to worry about paths.

Also, for some reason when I specified my bundle in the `package.json`, it didn't concat the `staticsRoot`. This is what I had to use to make it work `[{"outputFile": "bundle.js", "files": ["statics/js/main.js"]}]`